images/iqsharp-base/Dockerfile

```
@@ -101,6 +103,14 @@ RUN chown ${USER} -R ${LOCAL_PACKAGES}/ && \
 # Install all wheels from the build context.
 RUN pip install $(ls ${LOCAL_PACKAGES}/wheels/*.whl)
 
+# Get the Azure CLI tool
+ENV AZURE_CLI_VERSION "2.14.2"
+
+RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
+
+# Get the az quantum extension
+RUN az extension add --source https://msquantumpublic.blob.core.windows.net/az-quantum-cli/quantum-latest-py3-none-any.whl
```

Based on the CI failure, looks like `--yes` may need to be added here:
```suggestion
RUN az extension add --yes --source https://msquantumpublic.blob.core.windows.net/az-quantum-cli/quantum-latest-py3-none-any.whl
```

---

Thanks!, you're right... Just added the option in the latest commit.